cfme/tests/automate/test_file_import.py

```diff
@@ -90,3 +91,59 @@ def test_upload_blank_file(appliance, upload_file):
             view.import_file.upload_file.fill(datastore.file_path)
             view.import_file.upload.click()
             view.flash.assert_message("Error: import processing failed: domain: *")
+
+
+@pytest.fixture
+def setup_automate_model(appliance):
+    """This fixture creates domain, namespace, klass"""
+    # names and display names of the domain, namespace, klass needs to be static to match with newly
+    # imported datastore.
+    domain = appliance.collections.domains.create(
+        name="bz_1440226",
+        description=fauxfactory.gen_alpha(),
+        enabled=True)
+
+    namespace = domain.namespaces.create(
+        name="test_name",
+        description=fauxfactory.gen_alpha()
+    )
+
+    klass = namespace.classes.create(
+        name="test_class",
+        display_name="test_class_display",
+        description=fauxfactory.gen_alpha()
+    )
+    yield domain, namespace, klass
+    klass.delete_if_exists()
+    namespace.delete_if_exists()
+    domain.delete_if_exists()
+
+
+@pytest.mark.meta(automates=[1440226])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1440226.zip", "bz_1440226", None)],
+    ids=["datastore_update"],
+)
+def test_automate_import_attributes_updated(setup_automate_model, import_datastore, import_data):
```

Comment:
Why are we creating domain , namespace and klass ? You are importing them using "bz_1440226.zip" right ?

Reply:
Yes. This test case scenario is related to updating namespace and class description. Which needs existing datastore created and then importing same datastore with different description of namespace and class.